specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+    },
+    "SubscriptionUnderManagementGroupProperties": {
+      "description": "The generic properties of subscription under a management group.",
+      "type": "object",
+      "properties": {
+        "tenant": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the subscription. For example, 00000000-0000-0000-0000-000000000000"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the subscription."
+        },
+        "parent": {
+          "title": "Parent",
+          "$ref": "#/definitions/DescendantParentGroupInfo",
+          "x-nullable": true
+        },
+        "state": {
+          "type": "string",
+          "description": "The state of the subscription."
+        }
+      }
+    },
+    "ManagementGroup": {
+      "description": "The management group details.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, Microsoft.Management/managementGroups",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the management group. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementGroupProperties"
+        }
+      }
+    },
+    "ManagementGroupProperties": {
+      "description": "The generic properties of a management group.",
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the management group. For example, 00000000-0000-0000-0000-000000000000"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the management group."
+        },
+        "details": {
+          "title": "Details",
+          "$ref": "#/definitions/ManagementGroupDetails"
+        },
+        "children": {
+          "description": "The list of children.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupChildInfo"
+          }
+        }
+      }
+    },
+    "ManagementGroupDetails": {
+      "description": "The details of a management group.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "number",
+          "format": "int32",
+          "description": "The version number of the object."
+        },
+        "updatedTime": {
```

Comment:
>updatedTime [](http://example.com/codeflow?start=9&length=11)

updatedTime and updatedBy should be in systemData at the top level, and their definitions should come from v5 common types.